src/test/scenarios/attestation/output/src/attestation/azext_attestation/vendored_sdks/attestation/aio/_configuration_async.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         self.subscription_id = subscription_id
         self.api_version = "2018-09-01-preview"
         self.credential_scopes = ['https://management.azure.com/.default']
-        kwargs.setdefault('sdk_moniker', 'azure-mgmt-attestation/{}'.format(VERSION))
+        kwargs.setdefault('sdk_moniker', 'mgmt-attestation/{}'.format(VERSION))
```

Comment:
>mgmt-attestation [](start = 42, length = 16)

why would the azure- gone?

Reply:
Actually I am not sure. This is new python sdk change